packages/paste-website/src/pages/components/alert/index.mdx

```diff
@@ -16,7 +16,6 @@ import {Heading} from '@twilio-paste/heading';
 import {Paragraph} from '@twilio-paste/paragraph';
 import {DoDont, Do, Dont} from '../../../components/DoDont';
 import {Blockquote} from '../../../components/Blockquote';
-import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
```

Comment:
Note: I just removed this import because there are no Callouts on the Alert docs page (I noticed it when I was trying to find a Callout to copy/paste for the ChatLog page)